Source/MythicaEditor/Private/MythicaEditorSubsystem.cpp

```diff
@@ -1362,10 +1385,27 @@ void UMythicaEditorSubsystem::OnExecuteJobResponse(FHttpRequestPtr Request, FHtt
     SetJobState(RequestId, EMythicaJobState::Queued);
 }
 
-int UMythicaEditorSubsystem::CreateJob(const FString& JobDefId, const FMythicaParameters& Params, const FString& ImportPath)
+int UMythicaEditorSubsystem::CreateJob(const FString& JobDefId, const FMythicaParameters& Params, const FString& ImportPath, UMythicaComponent* Component)
 {
     int RequestId = NextRequestId++;
-    Jobs.Add(RequestId, { JobDefId, {}, Params, ImportPath });
+
+    FMythicaJob& Job = Jobs.Add(RequestId, { JobDefId, {}, Params, ImportPath });
+
+    Job.StartTime = FDateTime::Now();
+
+    if (IsValid(Component))
+    {
+        FString DisplayName = UKismetSystemLibrary::GetDisplayName(Component);
```

Comment:
I think we touched on this when we chatted already, but I don't think GetDisplayName is guaranteed to be unique. UMythicaComponent::ComponentGUID was my attempt to work around this.